SU-HKKU/cardinal_service_profile
# CSD-368 Hid ajax loader display to prevent visual shifting.

# READY FOR REVIEW

# Summary
- Hid ajax loader display to prevent visual shifting.

# Needed By (Date)
- 08/31

# Urgency
- low
# Steps to Test

1. Pull this PR
2. Clear cache
3. Sign onto your local and try favoriting items on the Opportunities list page. The `[heart] Saved` text should no longer shift.

# Associated Issues and/or People
- [CSD-368](https://stanfordits.atlassian.net/browse/CSD-368)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
